src/pages/en/getting-started.md

```diff
@@ -7,32 +7,40 @@ title: Getting Started
 description: A basic intro to Astro.
 i18nReady: true
 ---
-Static Site Generator  🚀  Bring your own Framework  🚀  Ship Less JavaScript
 
-:::tip
-Have an older project? Follow the [migration guide](/en/migrate/) to upgrade to v1.0 beta!
-:::
+#### What is Astro?
 
-## Try Astro
+Astro is an **all-in-one** **web framework** for building **fast,** **content-focused** websites. 
 
-We've made it as easy as possible to get started with Astro either in your browser, or on your machine!
+#### Key Features
 
-### Online Playgrounds
+- **Component Islands:** A new web architecture for building faster websites.
+- **Server-first API design:** Move expensive hydration off of your users' devices.
+- **Zero-JavaScript, by default:** No JS runtime overhead to slow you down.
+- **Edge-ready:** Deploy anywhere, even a global edge runtime like Deno or Cloudflare.
+- **Customizable:** Tailwind, MDX, and 100+ other integrations to choose from.
+- **Supports React, Preact, Svelte, Vue and all of your favorite UI frameworks!**
 
-Visit [astro.new](https://astro.new/) for the easiest way to "try before you buy." Choose from a variety of starter templates and start building a full, working version of Astro right in your browser!
+<!-- - **`client:visible` component loading:** If your user never sees it, it never loads. -->
+<!-- - **Image optimizations:** Astro's very own `<Image />` component. -->
+<!-- - **TypeScript support**  -->
+<!-- - **File-based routing:** Every file in the pages directory becomes a route. -->
 
-Or, **instantly launch our basic starter project** with just one click of a button:
+Sounds interesting? Check out our detailed [Why Astro](/en/concepts/why-astro/) breakdown to learn more.
+
+
+## Try Astro in your browser
+
+Visit [astro.new](https://astro.new/) and choose from a variety of templates to get started. Play around with a full, working version of Astro right in your browser!
 
 <div style="display: flex; flex-wrap: wrap; gap: 0.5rem;">
-    <Button href="https://astro.new/basics?on=codesandbox">Open in CodeSandbox</Button>
-    <Button href="https://astro.new/basics?on=stackblitz">Open in StackBlitz</Button>
+    <Button href="https://astro.new/basics?on=stackblitz">Quickstart!</Button>
+    <Button variant="outline" href="https://astro.new/">View all templates →</Button>
 </div>
 
-### Install Astro Locally
-
-Ready to install?
+## Start your first project
 
-Get a new project up and running locally in no time with our easy `create-astro` CLI wizard!
+Get a new Astro project up and running locally in no time with our easy `create-astro` CLI wizard!
 
 ```bash
```

Comment:
I tried but couldn't get very far! I think we'll need MDX for this:

```
<!-- didn't render properly, probably an md-astro bug -->
<Tabs client:visible sharedStore="ui-frameworks">
    <Fragment slot="tab.npm">npm</Fragment>
    <Fragment slot="panel.npm">
        <pre><code set:html={`# create a new project with npm\nnpm create astro@latest`} /></pre>
    </Fragment>
</Tabs>
```

Reply:
The Sharing State page uses an Astro component as a wrapper of the Tabs component, I think it's safe to add a package managers tab component, it can be really handy in some cases! 

You can copy and paste this and it will work perfectly (you just have to change the single quotes for backticks):
```html
// src/components/tabs/PackageManagerTabs.astro
---
import Tabs from './Tabs'
---

<Tabs client:visible sharedStore="package-managers">
  <Fragment slot="tab.npm">npm</Fragment>
  <Fragment slot="tab.pnpm">pnpm</Fragment>
  <Fragment slot="tab.yarn">Yarn</Fragment>

  <Fragment slot="panel.npm"><slot name="npm" /></Fragment>
  <Fragment slot="panel.pnpm"><slot name="pnpm" /></Fragment>
  <Fragment slot="panel.yarn"><slot name="yarn" /></Fragment>
</Tabs>
```
```html
---
// page.md
< !--- >
setup: |
  import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
---

<PackageManagerTabs>
  <Fragment slot="npm">
  '''shell
  npm create astro@latest
  '''
  </Fragment>
  <Fragment slot="pnpm">
  '''shell
  pnpm create astro@latest
  '''
  </Fragment>
  <Fragment slot="yarn">
  '''shell
  yarn create astro
  '''
  </Fragment>
</PackageManagerTabs>
```